client/onboarding-prototype/strings.tsx

```diff
@@ -73,6 +73,12 @@ export default {
 			'woocommerce-payments'
 		),
 	},
+	errors: {
+		generic: __( 'Please provide a response', 'woocommerce-payments' ),
+		email: __( 'Please provide a valid email', 'woocommerce-payments' ),
+		phone: __( 'Please provide a valid phone', 'woocommerce-payments' ),
```

Comment:
Nitpick: I think `Please provide a valid phone number` reads better here. What do you think?

Reply:
Definitely! I was in automatic mode adding the name of the field to the error 😄  Addressed in 0fcefdb